Could not GET vantiq-sdk from bintray

I’m facing below error during compile jdbcSource that seems related to Bintray service has now been sunset, according to [Bintray](https://bintray.com/). Please guide me how to get vantiq-sdk instead of Bintray/JCenter.

Error message of ./gradlew jdbcSource:assemble is as follows:

```
ubuntu@ip-172-31-38-144:~/extsrc/vantiq-extension-sources$ ./gradlew jdbcSource:assemble
 
> Configure project :
The property "dockerRegistryPassword" was not provided. This may affect publish operations.
 
> Task :jdbcSource:compileJava FAILED
 
FAILURE: Build failed with an exception.
 
* What went wrong:
Execution failed for task ':jdbcSource:compileJava'.
> Could not resolve all files for configuration ':jdbcSource:compileClasspath'.
   > Could not resolve io.vantiq:vantiq-sdk:1.0.17.
     Required by:
         project :jdbcSource
      > Could not resolve io.vantiq:vantiq-sdk:1.0.17.
         > Could not get resource 'https://dl.bintray.com/vantiq/maven/io/vantiq/vantiq-sdk/1.0.17/vantiq-sdk-1.0.17.pom'.
            > Could not GET 'https://dl.bintray.com/vantiq/maven/io/vantiq/vantiq-sdk/1.0.17/vantiq-sdk-1.0.17.pom'. Received status code 403 from server: Forbidden
 
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
 
* Get more help at https://help.gradle.org
 
Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.0/userguide/command_line_interface.html#sec:command_line_warnings
 
BUILD FAILED in 6s
3 actionable tasks: 1 executed, 2 up-to-date
```

--- a/CSVSource/build.gradle
+++ b/CSVSource/build.gradle
@@ -44,7 +44,7 @@ dependencies {
     compile "org.apache.logging.log4j:log4j-slf4j-impl:2.11.0"
 
     // Used in tests
-    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
+    testCompile "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
     
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/CSVSource/build.gradle
+++ b/CSVSource/build.gradle
@@ -9,11 +9,6 @@ sourceCompatibility = 1.8
 repositories {
     mavenCentral()
     jcenter()
-    
-    // Used for checking VANTIQ date format behavior with VANTIQ SDK
-    maven {
-    	url "https://dl.bintray.com/vantiq/maven"
-    }
 }
 
 mainClassName = 'io.vantiq.extsrc.CSVSource.CSVMain'

--- a/CSVSource/build.gradle
+++ b/CSVSource/build.gradle
@@ -42,10 +42,9 @@ dependencies {
     
     compile "org.slf4j:slf4j-api:1.7.25"
     compile "org.apache.logging.log4j:log4j-slf4j-impl:2.11.0"
-    
-      
-    // Used for checking VANTIQ date format behavior with VANTIQ SDK
-    compile 'io.vantiq:vantiq-sdk:1.0.27'
+
+    // Used in tests
+    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
     
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/EasyModbusSource/build.gradle
+++ b/EasyModbusSource/build.gradle
@@ -49,9 +49,9 @@ dependencies {
     compile "org.apache.logging.log4j:log4j-slf4j-impl:2.11.0"
     
     compile files("${System.env.EASY_MODBUS_LOC}/EasyModbusJava.jar")
-        
-    // Used for checking VANTIQ date format behavior with VANTIQ SDK
-    compile 'io.vantiq:vantiq-sdk:1.0.27'
+
+    // Used in tests
+    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
     
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/EasyModbusSource/build.gradle
+++ b/EasyModbusSource/build.gradle
@@ -10,11 +10,6 @@ sourceCompatibility = 1.8
 repositories {
     mavenCentral()
     jcenter()
-    
-    // Used for checking VANTIQ date format behavior with VANTIQ SDK
-    maven {
-    	url "https://dl.bintray.com/vantiq/maven"
-    }
 }
 
 mainClassName = 'io.vantiq.extsrc.EasyModbusSource.EasyModbusMain'

--- a/EasyModbusSource/build.gradle
+++ b/EasyModbusSource/build.gradle
@@ -51,7 +51,7 @@ dependencies {
     compile files("${System.env.EASY_MODBUS_LOC}/EasyModbusJava.jar")
 
     // Used in tests
-    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
+    testCompile "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
     
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/build.gradle
+++ b/build.gradle
@@ -42,6 +42,7 @@ group 'io.vantiq'
 ext {
     slf4jApiVersion = '1.7.25'
     eclipseMiloVersion = '0.2.1'
+    vantiqSDKVersion = "1.0.28"
 }
 
 

--- a/extjsdk/build.gradle
+++ b/extjsdk/build.gradle
@@ -81,7 +81,7 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
 
     // Used in tests
-    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
+    testCompile "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
 
     // Used to create EvictingQueue for failed messages queue
     implementation 'com.google.guava:guava:23.0'

--- a/extjsdk/build.gradle
+++ b/extjsdk/build.gradle
@@ -33,9 +33,7 @@ jar.from(".") {
 }
 
 repositories {
-    maven {
-        url "https://dl.bintray.com/vantiq/maven"
-    }
+    mavenCentral()
 }
 
 configurations {

--- a/extjsdk/build.gradle
+++ b/extjsdk/build.gradle
@@ -79,8 +79,9 @@ dependencies {
     compile "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
 
     testCompile group: 'junit', name: 'junit', version: '4.12'
-    // Used for uploading documents (images) to VANTIQ with VANTIQ SDK
-    testCompile 'io.vantiq:vantiq-sdk:1.0.24'
+
+    // Used in tests
+    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
 
     // Used to create EvictingQueue for failed messages queue
     implementation 'com.google.guava:guava:23.0'

--- a/jdbcSource/build.gradle
+++ b/jdbcSource/build.gradle
@@ -9,11 +9,6 @@ sourceCompatibility = 1.8
 repositories {
     mavenCentral()
     jcenter()
-    
-    // Used for checking VANTIQ date format behavior with VANTIQ SDK
-    maven {
-    	url "https://dl.bintray.com/vantiq/maven"
-    }
 }
 
 mainClassName = 'io.vantiq.extsrc.jdbcSource.JDBCMain'

--- a/jdbcSource/build.gradle
+++ b/jdbcSource/build.gradle
@@ -75,7 +75,7 @@ dependencies {
     runtime files("${System.env.JDBC_DRIVER_LOC}")
     
     // Used in tests that check VANTIQ date format behavior
-    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
+    testCompile "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
     
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/jdbcSource/build.gradle
+++ b/jdbcSource/build.gradle
@@ -74,7 +74,7 @@ dependencies {
     //  Compiling relevant JDBC Driver .jar file
     runtime files("${System.env.JDBC_DRIVER_LOC}")
     
-    // Used for test that checks VANTIQ date format behavior with VANTIQ SDK
+    // Used in tests that check VANTIQ date format behavior
     testCompile 'io.vantiq:vantiq-sdk:1.0.28'
     
     // Use JUnit test framework

--- a/jdbcSource/build.gradle
+++ b/jdbcSource/build.gradle
@@ -74,8 +74,8 @@ dependencies {
     //  Compiling relevant JDBC Driver .jar file
     runtime files("${System.env.JDBC_DRIVER_LOC}")
     
-    // Used for checking VANTIQ date format behavior with VANTIQ SDK
-    compile 'io.vantiq:vantiq-sdk:1.0.17'
+    // Used for test that checks VANTIQ date format behavior with VANTIQ SDK
+    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
     
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/jmsSource/build.gradle
+++ b/jmsSource/build.gradle
@@ -65,7 +65,7 @@ dependencies {
     compile files("${System.env.JMS_DRIVER_LOC}")
 
     // Used for testing extension source in VANTIQ
-    testCompile 'io.vantiq:vantiq-sdk:1.0.17'
+    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
 
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/jmsSource/build.gradle
+++ b/jmsSource/build.gradle
@@ -65,7 +65,7 @@ dependencies {
     compile files("${System.env.JMS_DRIVER_LOC}")
 
     // Used for testing extension source in VANTIQ
-    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
+    testCompile "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
 
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/jmsSource/build.gradle
+++ b/jmsSource/build.gradle
@@ -9,11 +9,6 @@ sourceCompatibility = 1.8
 repositories {
     mavenCentral()
     jcenter()
-
-    // Used for testing extension source in VANTIQ
-    maven {
-        url "https://dl.bintray.com/vantiq/maven"
-    }
 }
 
 mainClassName = 'io.vantiq.extsrc.jmsSource.JMSMain'

--- a/objectRecognitionSource/build.gradle
+++ b/objectRecognitionSource/build.gradle
@@ -254,7 +254,7 @@ dependencies {
     }
     
     // Used for uploading documents (images) to VANTIQ with VANTIQ SDK
-    compile 'io.vantiq:vantiq-sdk:1.0.28'
+    compile "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
 
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/objectRecognitionSource/build.gradle
+++ b/objectRecognitionSource/build.gradle
@@ -254,7 +254,7 @@ dependencies {
     }
     
     // Used for uploading documents (images) to VANTIQ with VANTIQ SDK
-    compile 'io.vantiq:vantiq-sdk:1.0.24'
+    compile 'io.vantiq:vantiq-sdk:1.0.28'
 
     // Use JUnit test framework
     testCompile 'junit:junit:4.12'

--- a/objectRecognitionSource/build.gradle
+++ b/objectRecognitionSource/build.gradle
@@ -12,11 +12,6 @@ sourceCompatibility = 1.8
 repositories {
     mavenCentral()
     jcenter()
-    
-    // Used for uploading documents (images) to VANTIQ with VANTIQ SDK
-    maven {
-    	url "https://dl.bintray.com/vantiq/maven"
-    }
 
     maven {
         url "http://vantiqmaven.s3.amazonaws.com/"

